sgkit/__init__.py

```diff
@@ -1,7 +1,7 @@
-from .api import (  # noqa: F401
+from .api import DIM_SAMPLE  # noqa: F401
+from .api import (
     DIM_ALLELE,
     DIM_PLOIDY,
-    DIM_SAMPLE,
```

Comment:
Nit: why did this change?

Reply:
must have been isort doing something weird, reverted to the original